docs/docs-content/clusters/data-center/nutanix/create-manage-nutanix-cluster.md

```diff
@@ -0,0 +1,119 @@
+description: "Learn how to create and manage Nutanix clusters in Palette."
+hide_table_of_contents: false
+tags: ["data center", "nutanix"]
+---
+
+Palette supports creating and managing Kubernetes clusters deployed to a Nutanix infrastructure environment. This section guides you in creating a Kubernetes cluster in a Nutanix cloud managed by Palette.
+
+## Prerequisites
+
+- A Nutanix cloud account added to Palette. Refer to [Add Nutanix Cloud Account](add-nutanix-cloud-account.md).
+
+<!-- A Nutanix Prism Central cloud account. For more information, review [Add Nutanix Cloud Account](add-nutanix-cloud-account.md).  -->
+
+- A Nutanix Private Cloud Gateway (PCG) deployed. For guidance, review [Install Private Cloud Gateway](install-pcg.md).
+
+- An infrastructure cluster profile created for the Nutanix cloud. For guidance on creating a profile, refer to [Create an Infrastructure Profile](../../../profiles/cluster-profiles/create-cluster-profiles/create-infrastructure-profile.md). At the **Cloud Type** step of profile creation, select **Nutanix** listed under the **Tech Preview**. Select the custom packs provided at the OS and Kubernetes layers. Palette provides out-of-the-box packs for the network and storage profile layers, including the **Nutanix CSI** storage pack.
+
+
+## Deploy a Nutanix Cluster
+
+Use the following steps to deploy a Kubernetes cluster in Nutanix.
+
+1. Log in to [Palette](https://console.spectrocloud.com/).
+
+2. From the left **Main Menu** select **Clusters**.
+
+3. Click on **Add New Cluster** and select **Deploy New Cluster** on the next page that Palette displays. 
+
+4. Select **Nutanix** and click the **Start Nutanix Configuration** button.
+
+5. Fill out the following basic information, and click **Next** to continue.
+
+  | **Field** | **Description** |
+  |-----------|-----------------|
+  | **Cluster Name**| A custom name for the cluster. |
+  | **Description**| Use the description to provide context about the cluster.|
+  | **Tags**| Assign any desired cluster tags. Tags on a cluster are propagated to the Virtual Machines (VMs) deployed to the target environments.|
+  | **Cloud Account** | Select your Nutanix account from the **drop-down Menu**. |
+
+6. Select the Nutanix cluster profile you created and click **Next**. Palette displays the profile layers.
+
+7. Review profile layers, leaving the OS and Kubernetes packs empty, and customize parameters as desired in the YAML files that display when you select the network and storage layers. Click **Next** when you are done.
+
+8. In the Cluster configuration YAML file that Palette displays, edit the file to replace variables within curly braces listed in the table below with values that apply to your Nutanix cloud environment, and make any adjustments to configure your cluster. Click **Next** when you are done.
+
+  | **Variable** | **Description** |
+  |--------------|-----------------|
+  | `${CLUSTER_NAME}`| The name of the Nutanix workload cluster. |
```

Comment:
Please add a comment that it must be the exact same name as the one created in palette in step 5

Reply:
Added the text in bold: The name of the Nutanix workload cluster. **Use the same cluster name you specified in step 5**.

Additionally, added the text in bold to the cluster Name description in step 5:
A custom name for the cluster. **Use this cluster name for `${CLUSTER_NAME}` variable in the YAML configuration files.**